apps/docs/app/ui/components/overview/Overview.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import { allComponents } from "@/.contentlayer/generated";
+import Image from "next/image";
+import "./overview.css";
+
+const Overview = () => {
+    // Get unique categories
+    const categories = Array.from(new Set(allComponents.map(component => component.category)));
+
+    return (
+        <div className="hd-component-overview-wrapper">
+            {categories.map(category => (
+                <div className="hd-component-overview-category" key={category}>
+                    <h2 className="hd-component-overview-category__title">{category}</h2>
```

Comment:
You should use the Title component 
https://github.com/gsoft-inc/wl-hopper/blob/main/apps/docs/components/title/Title.tsx